webpack.addon.config.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+const WebpackNotifierPlugin = require("webpack-notifier");
+const webpack = require("webpack");
+const path = require("path");
+
+module.exports = {
+  entry: {
+    "vendor": path.join(__dirname, "./addon-src/vendor.js")
```

Comment:
eh, `path` knows what to do to make it right for the platform. the initial "./" is kinda unnecessary though